cmd/contour/serve.go

```diff
@@ -99,7 +99,7 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 	serve.Flag("root-namespaces", "Restrict contour to searching these namespaces for root ingress routes.").StringVar(&ctx.rootNamespaces)
 
 	serve.Flag("ingress-class-name", "Contour IngressClass name.").StringVar(&ctx.ingressClass)
-
+	serve.Flag("ingress-status-address", "Addresses Envoy deployment is reachable at from outside the cluster.").StringVar(&ctx.ingressStatusAddress)
```

Comment:
s/Addresses/Address/

but maybe consider something  "Address to set in Ingress object status."?